src/oxygen/robot_interface.py

```diff
@@ -257,7 +257,8 @@ def spawn_robot_keyword(self,
 
         for message in messages:
             if message:
-                robot_keyword.messages.append(RobotResultMessage(message))
+                ishtml = message[:6] == '*HTML*'
+                robot_keyword.messages.append(RobotResultMessage(message,html=ishtml))
```

Comment:
What do you think: should we strip away the `*HTML*` from the beginning of the message, or not?